pkg/cmd/provisioning/identity_provider.go

```diff
@@ -321,8 +321,8 @@ func NewIdentityProviderSetup() *cobra.Command {
 		Run: identityProviderCmd,
 	}
 
-	identityProviderSetupCmd.PersistentFlags().StringVar(&CreateOpts.InfraName, "infra-name", "", "Name prefix for all created AWS resources")
-	identityProviderSetupCmd.MarkPersistentFlagRequired("infra-name")
+	identityProviderSetupCmd.PersistentFlags().StringVar(&CreateOpts.NamePrefix, "name-prefix", "", "Name prefix for all created AWS resources")
+	identityProviderSetupCmd.MarkPersistentFlagRequired("name-prefix")
```

Comment:
my personal preference is keeping infra name, and explaining that resource names use this as prefix and tags use this to mark owned

Reply:
We don't really know the infra name at this point though do we? It's generated during install and we're pre-install at this point. Cluster name perhaps but gotta watch length carefully.

Reply:
because the OIDC pieces can be shared across clusters, it seemed like there was a potential for confusion if we linked the cluster infra name to the OIDC infra resources. that's why i was pushing away from the 'infra-name' terminology.